sample/kotlin/src/fresco/res/layout/fragment_picture.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~ Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+  ~ This product includes software developed at Datadog (https://www.datadoghq.com/).
+  ~ Copyright 2016-Present Datadog, Inc.
+  -->
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+                                                   xmlns:app="http://schemas.android.com/apk/res-auto"
+                                                   xmlns:fresco="http://schemas.android.com/apk/res-auto"
+                                                   xmlns:tools="http://schemas.android.com/tools"
+                                                   android:layout_width="match_parent"
+                                                   android:layout_height="match_parent"
+                                                   android:padding="10dp"
+                                                   tools:context=".picture.PictureFragment">
+
+    <com.facebook.drawee.view.SimpleDraweeView
+        android:id="@+id/picture"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        fresco:placeholderImage="@drawable/ic_dd_icon_white"
+        fresco:actualImageScaleType="focusCrop"
+        fresco:placeholderImageScaleType="fitCenter"
+        fresco:viewAspectRatio="1"/>
```

Comment:
What is a `SimpleDraweeView` ? Doesn't Fresco work with regular Image Views ?

Reply:
Nope...they provide this extension class which you need to use in your layout, there's also a much more complicated way if you want to use it with an ImageView which implies a lot of boiler code, not very popular so I wanted to keep the popular approach.